src/plugins/discover/public/hooks/use_navigation_props.tsx

```diff
@@ -85,18 +83,14 @@ export const useNavigationProps = ({
     () => getCurrentBreadcrumb(history?.location?.search),
     [history?.location?.search]
   );
-  const contextSearchHash = useMemo(
-    () => getContextHash(columns, filterManager),
-    [columns, filterManager]
```

Comment:
This code just worked because columns ref changed ever time the state was updated. With the changes in this PR this is no longer the case, and so this caused a functional error, since the link to the context view / surrounding docs view was no longer until the columns were changed.